app/controllers/chargeback_controller.rb

```diff
@@ -197,6 +197,7 @@ def cb_rate_edit
           detail.metric = r[:metric]
           detail.chargeback_rate_detail_measure_id = r[:chargeback_rate_detail_measure_id]
           detail.chargeback_rate_detail_currency_id = r[:chargeback_rate_detail_currency_id]
+          @sb[:rate_details][:currency] = r.detail_currency.code
```

Comment:
is this a similar problem as below... are we querying the database here for the detail_currency?  Can we change line 187 above from:

```ruby
rate_details = rate.chargeback_rate_details
```

To:
```ruby
rate_details = rate.includes(:chargeback_rate_details => :detail_currency)
```